crates/runtime/src/helpers.c

```diff
@@ -1,5 +1,15 @@
 #include <setjmp.h>
 
+// Note that `sigsetjmp` and `siglongjmp` are used here to explicitly pass a 0
+// argument to `sigsetjmp` that we don't need to preserve the process signal
+// mask. This should make this call a bit faster b/c it doesn't need to touch
+// the kernel signal handling routines.
+#ifndef CFG_TARGET_OS_windows
+#define setjmp(buf) sigsetjmp(buf, 0)
```

Comment:
tiny nit, but would it be possible to choose other names for the macros rather than aliasing `setjmp`/`longjmp`/`jmp_buf`? This could be confusing otherwise; best to avoid surprising macro indirections :-)